src/styles/utils/mixins.scss

```diff
@@ -0,0 +1,11 @@
+@mixin hands-styles($hand-height, $hand-width) {
+  position: absolute;
+  top: setHandTopPosition($hand-height);
+  left: 50%;
+  transform: translateX(-50%);
```

Comment:
what do you think?
```suggestion
  top: 50%;
  left: 50%;
  transform: translateX(-50%, -50%);
```

Reply:
But I also have another question:
The animation description also has a transform property (rotate). And it seems that it rewrites other transform property (translate)  in the element's styles. So the rotation looks proper only if I duplicate  `translate` in animation. Is that the way it should be? Or there is another decision?